File: teethris-client-windows/network.py
from socketIO_client import SocketIO, LoggingNamespace

class Network:
    def __init__(self, incomming_message_callback):
        self.socketIO = SocketIO('borisjeltsin.azurewebsites.net', 80, LoggingNamespace)
        self._register_message_callback(incomming_message_callback)

    def _register_message_callback(self, callback):
        self.socketIO.on('chat message', callback)

    def send_message(self, message):
        self.socketIO.emit('chat message', message)

File: teethris.NET/Network.cs
using System;
using Quobject.SocketIoClientDotNet.Client;

namespace teethris.NET
{
    public class Network
    {
        public Network(){
            var socket = IO.Socket("http://borisjeltsin.azurewebsites.net/");
            socket.On(Socket.EVENT_CONNECT, () =>
            {
                socket.Emit("chat message", "hi");
            });

            socket.On("chat message", (data) =>
                {
                    Console.WriteLine(data);
                    socket.Disconnect();
                });
        }
    }
}